test/index.test.js

```diff
@@ -1222,4 +1224,85 @@ Previous tag: ''
       expect.assertions(1)
     })
   })
+
+  describe('with config-name input', () => {
+    it('loads from another config path', async () => {
+      /*
+        Mock
+        with:
+          config-name: 'config-name-input.yml'
+      */
+      process.env['INPUT_CONFIG-NAME'] = 'config-name-input.yml'
+
+      // Mock config request for file 'config-name-input.yml'
+      const getConfigScope = nock('https://api.github.com')
```

Comment:
I am assuming you didn't use `getConfigMock` because you wouldn't be able to do `getConfigScope.isDone()`? I've never used the `isDone` method so I'm not sure :)

Reply:
Actually, I didn't use `getConfigMock` because it statically requests `/repos/toolmantim/release-drafter-test-project/contents/.github/release-drafter.yml` and I actually want to assert that it requests the correct config file (based on the input).
`getConfigScope.isDone()` is just to check that the request was actually performed.
Basically I'm asserting two things: the config file request and the resulting output body for that configuration.